.github/workflows/ci.yaml

```diff
@@ -2,8 +2,7 @@ name: Test
 on:
   pull_request:
   push:
-    branches:
-      - master
+
```

Comment:
Forgot to ask about this in my previous review: why was this removed?

Reply:
To run the CI on all pushes. I can revert it if you'd like, but it's useful if you're working in a branch to have the CI check everything even before making a MR.

Reply:
It's probably best to keep this limited to master, otherwise pushes in all forks will start triggering these builds. The CI build submits coverage and requires credentials setup, so it will start failing in forks, presumably.

Personally, I often push updates in my own beets fork and I'm kind of glad that I'm not receiving notification about failed linting builds and such.

Reply:
Given your use case, a potential solution could be to open a draft PR while you're working on it?

Reply:
I dropped the commit from the PR.

I really don't mind keeping a commit just for my fork and removing it before making a PR, but a couple things don't match my experience:
- there's an extra step before the CI runs in a fork, you have to manually go into the actions tab and click "yes I understand my workflows"
- Codecov works out of the (beet)box: https://app.codecov.io/gh/beetbox/beets/tree/ThinkChaos%2Fbeets%3Afix%2Fsave-album-flexible-fields/beets
- I don't get any notifications for failed CI, so there's likely a user preference somewhere

Reply:
Good point about the non PR branches, they don't show up in the UI AFAICT, and they also don't fail